src/arraymancer/tensor/aggregate.nim

```diff
@@ -174,7 +174,7 @@ proc argmax_max*[T](t: Tensor[T], axis: int): tuple[indices: Tensor[int], maxes:
   ##                                     [1],
   ##                                     [1]].toTensor
 
-  assert axis in {0, 1}, "Only 1D and 2D tensors are supported at the moment for argmax"
+  # assert axis in {0, 1}, "Only 1D and 2D tensors are supported at the moment for argmax"
```

Comment:
This needs both a changelog entry and a test imo. Not sure if this should be part of this PR.

Reply:
Hmm, I wondered that as well but considering the issue associated to the assert in the comment has been closed I treated this assert as a deprecated warning that wasn't relevant anymore.

+1 for changelog & tests.

Reply:
I'll add test and changelog this week I hope. I just had stuff to get done so couldn't take the time to take care of it.